packages/website/src/features/Docs/DocsCannonfilesPage.tsx

```diff
@@ -395,7 +395,15 @@ export const DocsCannonfilesPage: FC = () => {
                     #
                   </Link>
                 </Heading>
-                <Text color="gray.400">Coming soon.</Text>
+                <Text color="gray.400">
+                  Primary source of cannon package information which contains
+                  package definition and on-chain state data derived changes
+                  made by defined steps in the cannonfile definition. Deployment
+                  data is stored on IPFS and is locally stored in your
+                  filesystem in the default storage location
+                  ~/.local/share/cannon/tags or the storage location defined by
```

Comment:
Some code styles for legibility:
```suggestion
                  <Code>~/.local/share/cannon/tags</Code> or the storage location defined by
```